rollouts/api/v1alpha1/rollout_types.go

```diff
@@ -116,8 +116,13 @@ type StrategyRollingUpdate struct {
 // StrategyProgressive defines the progressive rollout strategy to use.
 type StrategyProgressive struct {
 	// Reference of ProgressiveRolloutStrategy to use.
-	Name      string `json:"name"`
-	Namespace string `json:"namespace"`
+	Name           string         `json:"name"`
+	Namespace      string         `json:"namespace"`
+	PauseAfterWave PauseAfterWave `json:"pauseAfterWave,omitempty"`
```

Comment:
we should add small explanation about the behavior :)

Since this is optional. If not specified, it is assumed to be automatic progression between the waves ?

Reply:
Added a description to the field to help explain the behavior in the latest commit.

Currently the field is required (unlike the pre pull request code) to avoid the distraction at the moment in specifying what happens when its not defined, and any defaulting logic that is needed.